src/waveform/renderers/glslwaveformrenderersignal.cpp

```diff
@@ -45,7 +45,7 @@ bool GLSLWaveformRendererSignal::loadShaders() {
     m_frameShaderProgram->removeAllShaders();
 
     if (!m_frameShaderProgram->addShaderFromSourceFile(
-            QGLShader::Vertex, ":shaders/passthrough.vert")) {
+            QGLShader::Vertex, ":/shaders/passthrough.vert")) {
```

Comment:
Maybe this was the cause of https://bugs.launchpad.net/mixxx/+bug/1872172 ?

Reply:
Not sure, but this only affects the glsl one (although I believe currently that's the default renderer in new setups)